lib/Activity.py

```diff
@@ -1050,9 +1050,11 @@ def get_nested_dict(dictarg, keyname):
 
     def watch_next_wf(self, sessionid):
         while True:
-            wfid = self.get_next_workflow(sessionid)
+            wfid,session_status = self.get_next_workflow(sessionid)
             if not wfid:
                 self.config.logger.debug(f"No more workflows found for session {sessionid}.")
+                if session_status == "debug":
```

Comment:
Should this line be
```
if wfid.session_status == "debug"
```

Reply:
Hi , wfid contains only workflow id and wfid , sssion_status are 2 seperate variables.
https://github.com/Cray-HPE/iuf-cli/blob/CASMTRIAGE-6716_partial_workflows/lib/Activity.py#L490